src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -98,8 +97,8 @@
     // Indicates the node.js version installed by nvm to set the PATH variable to point to
     private static final String NODE_VERSION = "v12.16.3";
     private static final String OTP_GRAPH_FILENAME = "Graph.obj";
+    public static final String OTP_RUNNER_BRANCH = "router-folder-downloads";
```

Comment:
Does this branch need to change? Probably needs a FIXME if it's not set to master?